cypress/integration/elements/LeftNav.js

```diff
@@ -47,13 +47,31 @@ class LeftNav{
         }
     }
 
-    getOpenToWorkspaceButton(){
-        return cy.get('[data-test=open-document-button]');
+    getOpenToWorkspaceButton(index){
+        return cy.get('#leftNavContainer' + index).find('[data-test=open-document-button]');
```

Comment:
@eireland @kevinsantos-cc Instead of just pressing the button (since there might be multiple open buttons), we find the container element using the correct index tied to the workspace name and then click the button inside the container.  How to we feel about using the element id here?